ksqldb-engine/src/test/java/io/confluent/ksql/internal/StorageUtilizationMetricsReporterTest.java

```diff
@@ -94,17 +93,6 @@ public void shouldAddNodeMetricsOnConfigure() throws IOException {
     assertThat((Double) pctUsedValue, greaterThan(0.0));
   }
 
-  @Test
-  public void shouldOnlyAddNodeMetricsOnce() {
-    // Given:
-
-    // When:
-    listener.configure(ImmutableMap.of("state.dir", "/tmp//"));
-
-    // Then:
-    // did not throw error for duplicate metrics
-  }
-
```

Comment:
I think at some point in the past, this was verifying that two listeners wouldn't try to register the same metrics in the static registry. At the time of this change, though, I guess the implementation of the listener had changed, and this test was actually verifying nothing. This is a good reminder to add some kind of positive check to such tests to make sure they're really validating what they think they are.

Regardless, the whole test is a truism now that we don't have a static registry and additionally now that sensor registration is idempotent.